source/linear-algebra/source/05-GT/05.ptx

```diff
@@ -171,9 +174,11 @@
     </task>
     <task>
         <p>
-            The matrix <m>A</m> describes how the standard basis of <m>\IR^3</m> is transformed by the linear transformation <m>T</m>.
-            The matrix <m>M_\mathcal{B}AM_{\mathcal{B}}^{-1}</m> describes how <m>\cal{B}</m> is transformed (in <m>\mathcal{B}</m>-coordinates). 
-            Which of these two descriptions is easier for you to describe/understand/visualize and why?
+            The matrix <m>A</m> describes how <m>T</m> transforms the standard basis of <m>\IR^3</m>.
+            The matrix <m>M_\mathcal{B}AM_{\mathcal{B}}^{-1}</m> describes how <m>T</m> transforms the basis <m>\mathcal{B}</m> (in <m>\mathcal{B}</m>-coordinates).
+        </p>
+        <p>         
+            Which of these two descriptions is most helpful to you in describing/understanding/visualizing the transformation <m>T</m> and why?
```

Comment:
```suggestion
            Which of these two descriptions of <m>T</m> is most helpful to you in describing/understanding/visualizing the transformation <m>T</m> and why?
```

Reply:
I think this new version clarifies your intention. I'm not sure if the extra copy of T I added sounds redundant, but it seems to help a bit.